python/tskit/trees.py

```diff
@@ -10520,6 +10532,129 @@ def sample_nodes_by_ploidy(self, ploidy):
         sample_node_ids = sample_node_ids.reshape((num_samples_per_individual, ploidy))
         return sample_node_ids
 
+    def map_to_vcf_model(
+        self,
+        individuals=None,
+        ploidy=None,
+        name_metadata_key=None,
+        individual_names=None,
+        include_non_sample_nodes=None,
+    ):
+        """
+        Maps the sample nodes in this tree sequence to a representation suitable for
+        VCF output, using the individuals if present.
+
+        Creates a VcfModelMapping object that contains both the nodes-to-individual
+        mapping as a 2D array of (individuals, nodes) and the individual names. The
+        mapping is created by first checking if the tree sequence contains individuals.
+        If it does, the mapping is created using the individuals in the tree sequence.
+        Be default only the sample nodes of the individuals are included in the mapping,
+        unless `include_non_sample_nodes` is set to True, in which case all nodes
+        belonging to the individuals are included. Any individuals without any nodes
+        are ignored and not included in the mapping.
+        If no individuals are present, the mapping is created using only the sample nodes
+        and the specified ploidy.
+
+        If neither `name_metadata_key` nor `individual_names` is not specified, the
+        individual names are set to "tsk_{individual_id}" for each individual.
```

Comment:
Document the naming convention for the no-individuals case above.

Reply:
Fixed in 5e3e4011